src/container.c

```diff
@@ -22,6 +22,8 @@
 #include "syscall.h"
 #include "netlink.h"
 
+#define MAX_PBUF 512
```

Comment:
Please just use `PATH_MAX` defined in <limits.h>, as being done in `container_check_volume`.